components/page-header/index.jsx

```diff
@@ -52,13 +52,13 @@ const renderBreadcrumb = (h, breadcrumb) => {
 
 const renderTitle = (h, prefixCls, instance) => {
   const { avatar } = instance;
-  const title = getComponentFromProp(instance, 'title');
-  const subTitle = getComponentFromProp(instance, 'subTitle');
-  const tags = getComponentFromProp(instance, 'tags');
-  const extra = getComponentFromProp(instance, 'extra');
+  const title = getComponent(instance, 'title');
+  const subTitle = getComponent(instance, 'subTitle');
+  const tags = getComponent(instance, 'tags');
+  const extra = getComponent(instance, 'extra');
   const backIcon =
-    getComponentFromProp(instance, 'backIcon') !== undefined ? (
-      getComponentFromProp(instance, 'backIcon')
+    getComponent(instance, 'backIcon') !== undefined ? (
+      getComponent(instance, 'backIcon')
     ) : (
       <ArrowLeftOutlined />
     );
```

Comment:
const onBack = instance.$listeners.back;
to
const onBack = instance.$attrs.onBack;